src/iOS/Xamarin.iOS/Samples/GeometryEngine/BufferList/BufferList.cs

```diff
@@ -0,0 +1,273 @@
+                UIAlertController alertController = UIAlertController.Create("Geometry Engine Failed!", ex.Message, UIAlertControllerStyle.Alert);
+                alertController.AddAction(UIAlertAction.Create("OK", UIAlertActionStyle.Default, null));
+                PresentViewController(alertController, true, null);
+                return;
+            }
+        }
+
+        private void CreateLayout()
+        {
+
+            // Create a UITextView for the overall sample instructions.
+            _sampleInstructionUITextiew = new UITextView();
+            _sampleInstructionUITextiew.Text = "Tap on the map in several locations to create center map-points to generate buffer(s). You can " + 
+                "optionally change the buffer distance(in miles) by adjusting the value in the text field before each tap on the map. Then click " + 
+                "on the 'Create Buffer(s)' button. If the 'Union the buffer(s)' switch is 'on' the resulting output buffer will be one polygon "+
+                "(possibly multi - part). If the 'Union the buffer(s)' switch is 'off' the resulting output will have one buffer polygon per input map point.";
+            _sampleInstructionUITextiew.Font = UIFont.FromName("Helvetica", 9f);
+
+            // Create a UILabel for instructions.
+            _bufferInstructionsUILabel = new UILabel();
+            _bufferInstructionsUILabel.Text = "Buffer distance (miles):";
+            _bufferInstructionsUILabel.AdjustsFontSizeToFitWidth = true;
+            _bufferInstructionsUILabel.BackgroundColor = UIColor.White;
+
+            // Create a UITextFiled for the buffer value.
+            _bufferDistanceMilesUITextField = new UITextField();
+            _bufferDistanceMilesUITextField.Text = "10";
+            _bufferDistanceMilesUITextField.AdjustsFontSizeToFitWidth = true;
+            _bufferDistanceMilesUITextField.BackgroundColor = UIColor.White;
+            // - Allow pressing 'return' to dismiss the keyboard
+            _bufferDistanceMilesUITextField.ShouldReturn += (textField) => { textField.ResignFirstResponder(); return true; };
+
+            // Create a UILabel for instructions.
+            _unionBufferInstructionsUILabel = new UILabel();
+            _unionBufferInstructionsUILabel.Text = "Union the buffer(s):";
+            _unionBufferInstructionsUILabel.AdjustsFontSizeToFitWidth = true;
+            _unionBufferInstructionsUILabel.BackgroundColor = UIColor.White;
+
+            // Create a UISwitch for toggling the union of the buffer geometries.
+            _unionBufferUISwitch = new UISwitch();
+            _unionBufferUISwitch.On = true;
+            _unionBufferUISwitch.BackgroundColor = UIColor.White;
+
+            // Create a UIButton to create the buffers.
+            _theUnionBufferButton = new UIButton();
+            _theUnionBufferButton.SetTitle("Make Unioned Buffer", UIControlState.Normal);
```

Comment:
I recommend changing this to be just 'make buffer'. 'Make Union Buffer' is not accurate when the union switch is unchecked.

Reply:
Note: also applies to Android.